src/hooks/requests/useUsers.ts

```diff
@@ -138,13 +138,31 @@ export const useDownloadUsers = () => {
   }
 }
 
-export const useArchiveUser = ({ userId }: { userId?: string }) => {
-  const { mutate: archiveUser, isLoading } = useMutation({
-    mutationKey: ['users', userId],
-    mutationFn: () => {
-      return apiClient.post(endpoints.ARCHIVE_USER, {
-        institution_user_id: userId,
-      })
+export const useArchiveUser = ({
+  institution_user_id,
+}: {
+  institution_user_id?: string
+}) => {
+  const queryClient = useQueryClient()
+  const { mutateAsync: archiveUser, isLoading } = useMutation({
+    mutationKey: ['users', institution_user_id],
+    mutationFn: async () =>
+      apiClient.post(endpoints.ARCHIVE_USER, institution_user_id),
```

Comment:
**[blocker]** (potentially)

Did you test this ? I don't think this will work, data structure seems to be wrong